src/mpsc/errors.rs

```diff
@@ -1,6 +1,26 @@
 //! Errors returned by channels.
 use core::fmt;
 
+/// Error returned by the [`Sender::try_send`] or [`Sender::try_send_ref`] (and
+/// [`StaticSender::try_send`]/[`StaticSender::try_send_ref`]) methods.
+///
+/// [`Sender::try_send`]: super::Sender::try_send
+/// [`Sender::try_send_ref`]: super::Sender::try_send_ref
+/// [`StaticSender::try_send`]: super::StaticSender::try_send
+/// [`StaticSender::try_send_ref`]: super::StaticSender::try_send_ref
```

Comment:
looks like this was copied from the `try_send` docs --- i think it should reference the `send_timeout` methods instead
```suggestion
/// Error returned by the [`Sender::send_timeout`] or [`Sender::send_ref_timeout`] 
/// (and [`StaticSender::send_timeout`]/[`StaticSender::send_ref_timeout`]) methods.
///
/// [`Sender::send_timeout`]: super::Sender::send_timeout
/// [`Sender::send_ref_timeout`]: super::Sender::send_ref_timeout
/// [`StaticSender::send_timeout`]: super::StaticSender::send_timeout
/// [`StaticSender::send_ref_timeout`]: super::StaticSender::send_ref_timeout
```